test/bdd/features/query.feature

```diff
@@ -0,0 +1,10 @@
+Feature: Querying assets on the DKG
+
+    Scenario: Querying the assertion by keyword on the DKG
+        Given query request is sent for the keyword
```

Comment:
You should look at a scenario as unit test. You need to prepare the network for each, and always use the same sentences. You can take a look at release.feature. Also each feature file should have background step defined (setting up blockchain and bootstrap node - again look at feature.release).